### PR TITLE
Add EconomyPlus Support!

### PR DESCRIPTION
**Pull request by ImagicalGamer**

### CODE REVIEW

**Comment by XFuryMCPE** [comment]
you have code issues.

**Comment by ImagicalGamer** [comment]
@XFuryMCPE ?

**Comment by Pogogo007** [comment]
Was this tested?

**Comment by ImagicalGamer** [comment]
Yes @Pogogo007 I tested it on my server

File: src/hud/Main.php [hunk 1]
```diff
@@ -42,6 +42,29 @@ public function onEnable(){
         $this->hudOff = new Config($this->getDataFolder() . "hudOff.yml",Config::YAML);
     }
     
+    public function economy(){
+        if($this->getConfig()->get("enable-economy-support") == false){
+            $this->money = null
```
Comment (XFuryMCPE) [inline src/hud/Main.php]: Missing ;, crashes

**Comment by Pogogo007** [comment]
ThAt was already fixed

**Pull request merged**